docs/api/render.md

```diff
@@ -11,7 +11,7 @@ disservice.
 
 For the purposes of this documentation, we will refer to Cheerio's constructor as
 `CheerioWrapper`, which is to say that it is analogous to our `ReactWrapper` and `ShallowWrapper`
-constructors.
+constructors. [Here](https://github.com/cheeriojs/cheerio#api) is a link to the API available from a `CheerioWrapper` instance.
```

Comment:
Can we rephrase this to:

> You can reference the [Cheerio API docs](https://github.com/cheeriojs/cheerio#api) for methods available on a `CheerioWrapper` instance.

I'm not a big fan of providing links with "here" if we can integrate it into a natural sentence otherwise